schemas/dto/recordBatch.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of record DTO",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "records": {
+      "description": "List of records",
```

Comment:
List of saved records

Reply:
Do you suggest me to create a separate schema for update (if it's needed) and write there "List of updated records"? Schema per operation?

Reply:
List of successfully processed records?